### PR TITLE
Containers bump to Bookworm

### PR DESCRIPTION
**Pull request by andreriesco**

### CODE REVIEW

**Comment by andreriesco** [comment]
Please review @microhobby

**Comment by microhobby** [comment]
@andreriesco please rebase it with `dev`.
Now that we have `bookwork` and `bullseye` all new changes has to be merged on `dev` first. Only fixes that are specific for `bullseye` that will be merged directly to `bullseye`.

**Review by microhobby — CHANGES_REQUESTED** [review]
@andreriesco please rebase it

File: cppQML/docker-compose.yml [hunk 1]
```diff
@@ -82,10 +79,8 @@ services:
       - CAP_SYS_TTY_CONFIG
     # Add device access rights through cgroup...
     device_cgroup_rules:
-      # ... for tty0
-      - "c 4:0 rmw"
-      # ... for tty7
-      - "c 4:7 rmw"
+      # ... for tty
+      - "c 4:* rmw"
```
Comment (microhobby) [inline cppQML/docker-compose.yml]: tty7 is not needed anymore?

File: dotnetAvalonia/docker-compose.yml [hunk 2]
```diff
@@ -55,12 +55,9 @@ services:
     ]
 
   weston:
-    image: torizonextras/weston${GPU}:2
+    image: torizon/weston${GPU}:3
     environment:
       - ACCEPT_FSL_EULA=1
-      - WESTON_ARGS=--continue-without-input
```
Comment (microhobby) [inline dotnetAvalonia/docker-compose.yml:61]: I'm pretty sure that this line is yet needed...

File: dotnetAvalonia/docker-compose.yml [hunk 3]
```diff
@@ -77,10 +74,8 @@ services:
       - CAP_SYS_TTY_CONFIG
     # Add device access rights through cgroup...
     device_cgroup_rules:
-      # ... for tty0
-      - "c 4:0 rmw"
-      # ... for tty7
```
Comment (microhobby) [inline dotnetAvalonia/docker-compose.yml:82]: tty7 is not needed anymore?

**Review by microhobby — COMMENTED** [review]
@andreriesco please take a look on the review, thanks

File: cppQML/docker-compose.yml (hunk 1 above)
Comment (andreriesco) [reply cppQML/docker-compose.yml]: `4:*` includes all tty ports I believe. With just 4:0 and 4:7 it was not working, I am not sure why.

File: dotnetAvalonia/docker-compose.yml (hunk 2 above)
Comment (andreriesco) [reply dotnetAvalonia/docker-compose.yml:61]: With this line Weston just hangs, I doesn't wait for the application container

File: dotnetAvalonia/docker-compose.yml (hunk 3 above)
Comment (andreriesco) [reply dotnetAvalonia/docker-compose.yml:82]: `4:*` includes all tty ports I believe. With just 4:0 and 4:7 it was not working, I am not sure why.

File: dotnetAvalonia/docker-compose.yml (hunk 2 above)
Comment (microhobby) [reply dotnetAvalonia/docker-compose.yml:61]: Did you tested without mouse, keyboard or touch screen connected?

Comment (microhobby) [reply dotnetAvalonia/docker-compose.yml:61]: @andreriesco commented with me internally that he tested without keyboard and touch screen and it works

File: dotnetAvalonia/docker-compose.yml (hunk 3 above)
Comment (andreriesco) [reply dotnetAvalonia/docker-compose.yml:82]: It was missing the tty1 port, now it is working

File: cppQML/docker-compose.yml (hunk 1 above)
Comment (andreriesco) [reply cppQML/docker-compose.yml]: It was missing the tty1 port, now it is working

File: assets/tasks/common.json [hunk 4]
```diff
@@ -612,32 +614,6 @@
                 "color": "terminal.ansiYellow"
             }
         },
-        {
-            "label": "pre-cleanup-x86",
```
Comment (microhobby) [inline assets/tasks/common.json:616]: I see that you remove it, maybe to change to `pre-cleanup-amd64`, but I cannot find the entry for it in the additions...

File: cppConsole/.vscode/tasks.json [hunk 5]
```diff
@@ -450,16 +450,16 @@
...
                 "apply-torizon-packages",
                 "copy-docker-compose",
-                "pre-cleanup-x86",
-                "build-makedir-x86",
-                "build-debug-x86",
-                "build-container-torizon-debug-x86",
-                "push-container-torizon-debug-x86",
-                "pull-container-torizon-debug-x86",
-                "run-container-torizon-debug-x86",
+                "pre-cleanup-amd64",
```
Comment (microhobby) [inline cppConsole/.vscode/tasks.json:456]: For example, here we are depending on `pre-cleanup-amd64` that does not exists

**Review by microhobby — CHANGES_REQUESTED** [review]

File: assets/tasks/common.json (hunk 4 above)
Comment (andreriesco) [reply assets/tasks/common.json:616]: This task was duplicated, so I just removed this one. https://github.com/andreriesco/vscode-torizon-templates/blob/tie-716/assets/tasks/common.json#L591

File: cppConsole/.vscode/tasks.json (hunk 5 above)
Comment (andreriesco) [reply cppConsole/.vscode/tasks.json:456]: This task was duplicated, so I just removed the duplicated one but it is still there. https://github.com/andreriesco/vscode-torizon-templates/blob/tie-716/assets/tasks/common.json#L591

**Review by microhobby — APPROVED** [review]

**Pull request merged**